hrm-service/migrations/20220324154600-create-CaseSections.js

```diff
@@ -0,0 +1,155 @@
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(
+      `CREATE SEQUENCE IF NOT EXISTS public."CaseSections_sectionId_seq"
+        INCREMENT 1
+        START 100000
+        MINVALUE 1
+        MAXVALUE 9223372036854775807
+        CACHE 1;`,
+    );
+    console.log("Sequence 'CaseSections_sectionId_seq' created");
+    await queryInterface.sequelize.query(
+      `ALTER SEQUENCE public."CaseSections_sectionId_seq" OWNER TO hrm`,
+    );
+    console.log("Sequence 'CaseSections_sectionId_seq' ownership altered.");
+    await queryInterface.sequelize.query(`CREATE TABLE IF NOT EXISTS public."CaseSections"
+      (
+          "caseId" integer NOT NULL,
+          "sectionType" text COLLATE pg_catalog."default" NOT NULL,
+          "sectionId" text COLLATE pg_catalog."default" NOT NULL DEFAULT nextval('"CaseSections_sectionId_seq"'::regclass),
```

Comment:
One more question and I'm done bothering (just asking to learn).
I see that you used `nextval('"CaseSections_sectionId_seq"'::regclass)::text` in the `CASE` statements below. What's the reason of this column being `text`, and is not necessary to specify the `::text` for the `DEFAULT` here?

Reply:
sectionId is 'text' because there's already a couple of different types of IDs out there, numbers & UUIDs. I wanted a field type that could take both, and whatever ID system we might use going forward

From testing the cast doesn't seem to be necessary, but I don't know why, the intricacies of PostgreSQL's type system and cvasting rules are not well known to me